src/Widgets/EligibilityModal/index.tsx

```diff
@@ -34,25 +35,63 @@ const EligibilityModal: FunctionComponent<Props> = ({
   const ModalComponent = isBigScreen ? DesktopModal : MobileModal
   const eligiblePlans = eligibilityPlans.filter((plan) => plan.eligible)
   const currentPlan = eligiblePlans[currentPlanIndex]
+  const intl = useIntl()
 
   const isSomePlanDeferred = eligibilityPlans.some(
     (plan) => plan.deferred_days > 0 || plan.deferred_months > 0,
   )
 
+  // Skip links definition for RGAA accessibility
+  // Towards the 3 main sections that users may want to quickly consult
+  const skipLinks = [
+    {
+      href: '#payment-plans',
+      labelId: 'skip-links.payment-plans',
+      defaultMessage: intl.formatMessage({
+        id: 'skip-links.payment-plans',
+        defaultMessage: 'Allez aux options de paiements',
+      }),
+    },
+    {
+      href: '#payment-info',
+      labelId: 'skip-links.payment-info',
+      defaultMessage: intl.formatMessage({
+        id: 'skip-links.payment-info',
+        defaultMessage: 'Allez aux informations de paiement',
+      }),
+    },
+    {
+      href: '#payment-schedule',
+      labelId: 'skip-links.payment-schedule',
+      defaultMessage: intl.formatMessage({
+        id: 'skip-links.payment-schedule',
+        defaultMessage: 'Allez au calendrier de paiements',
+      }),
+    },
+  ]
+
   return (
-    <Modal onClose={onClose} ariaHideApp={false} scrollable isOpen>
+    <Modal
+      onClose={onClose}
+      ariaHideApp={false}
+      scrollable
+      isOpen
+      aria-labelledby="eligibility-modal"
+      aria-describedby="control the user's eligibility for payment plans"
```

Comment:
I don't know if we have to push harder and translate this too ? Or can we accept to keep this in english ? (I think it just depends on if it's read by screen reader or not 🤔 )